thavalon-webapp/src/components/gameComponents/constants.ts

```diff
@@ -24,6 +24,31 @@ export enum Vote {
     Downvote = "Downvote",
 }
 
+/**
+ * An enum of all GameMessage types
+ */
+export enum GameMessageType {
+    Error = "Error",
```

Comment:
Error should probably be camel case also, to match all other gameMessageType values